e2e/wdio/platform/tests/time-picker.e2e-spec.ts

```diff
@@ -0,0 +1,114 @@
+import {
+    click,
+    doesItExist,
+    elementArray,
+    getAttributeByName,
+    getElementArrayLength,
+    getText,
+    getValue,
+    refreshPage,
+    scrollIntoView,
+    sendKeys,
+    setValue,
+    waitForElDisplayed,
+    waitForPresent
+} from '../../driver/wdio';
+import { time, text, defaultValidTime } from '../fixtures/testData/time-picker';
+import { TimePicker } from '../pages/time-picker.po';
+
+let timePickerPage: TimePicker;
+const {
+    activeTimePickerInput, timePickerButton, timePickerInput, timerExpanded,
+    activeTimePickerButton, errorBorder,
+    selectedHours, selectedMinutes, period,
+    navigationDownArrowButton, timeItem, setToNullButton, setValidTimeButton
+} = new TimePicker();
+
+describe('Time picker suite', function() {
+    timePickerPage = new TimePicker();
+
+    beforeAll(() => {
+        timePickerPage.open();
+    }, 1);
+
+    afterEach(() => {
+        refreshPage();
+        waitForPresent(timePickerInput);
+    }, 1);
+
+    it('Verify in all the form factor user is able to see the date picker button and input field ', () => {
+        const buttons = elementArray(timePickerButton);
+        const inputs = elementArray(timePickerInput);
+        expect(buttons.length).toEqual(inputs.length);
```

Comment:
why not use getElementArrayLength() and rename the variables to buttonsCount, inputsCount. then expect(buttonsCount).toEqual(inputsCount);